lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Model for Course Card.
+ */
+(function (define) {
+    'use strict';
+    define([
+            'backbone'
+        ], 
+        function (Backbone) {
+        return Backbone.Model.extend({
+            initialize: function(data) {
+                if (data){
+                    this.set({
+                        name: data.display_name,
+                        courseStart: data.course_start,
```

Comment:
I see what you're saying, I thought you meant not using a model at all. I agree that if we are not doing anything other than setting and getting from the model it makes sense to just use the default Backbone Model.